src/sdk/pynni/nni/nas/pytorch/nas_bench_201/nas_bench_201.py

```diff
@@ -0,0 +1,72 @@
+from collections import OrderedDict
+import torch.nn as nn
+from nni.nas.pytorch.mutables import LayerChoice
+
+from .nas_bench_201_ops import Pooling, ReLUConvBN, Zero, FactorizedReduce
+
+
+class NASBench201Cell(nn.Module):
+    """
+    Builtin cell structure of NAS Bench 201. One cell contains four nodes. The First node serves as an input node
+    accepting the output of the previous cell. And other nodes connect to all previous nodes with an edge that
+    represents an operation chosen from a set to transform the tensor from the source node to the target node.
+    Every node accepts all its inputs and adds them as its output.
+
+    Parameters
+    ---
+    cell_id: str
+        the name of this cell
+    C_in: int
+        the number of input channels
```

Comment:
could be more concrete here, for example, "the number of input channels of this cell"